### ISSUE TITLE
Missing the aria-label attribute of the element selector in the logic tab

### ISSUE THREAD

**Issue by tsv2013** [surveyjs/survey-creator#831]
See the https://surveyjs.answerdesk.io/ticket/details/t4332/survey-creator-survey-logic-accessibility-issue-ensure-every-form-element-has-a-label thread for more details
